lib/shape.ml

```diff
@@ -24,36 +25,59 @@ let draw_line x1 y1 x2 y2 = draw_poly_line [| (x1, y1); (x2, y2) |]
 let denormalize point =
   { x = point.x + canvas_mid.x; y = point.y + canvas_mid.y }
 
-let render_shape s =
+  let render_shape s =
   match s with
   | Circle circle ->
-      draw_circle (denormalize circle.c).x (denormalize circle.c).y
-        circle.radius
+      set_color black;
```

Comment:
This is done by default in the `init` function. Rather reset it back to default after the current rendering is done?

Reply:
Okay, got it.